internal/pkg/provision/providers/docker/node.go

```diff
@@ -109,9 +109,8 @@ func (p *provisioner) createNode(ctx context.Context, clusterReq provision.Clust
 
 	// Mutate the container configurations based on the node type.
 
-	switch nodeReq.Config.Machine().Type() {
-	case runtime.MachineTypeInit:
-		portsToOpen := []string{"50000:50000/tcp", "6443:6443/tcp"}
+	if nodeReq.Config.Machine().Type() == runtime.MachineTypeControlPlane {
```

Comment:
should it be both MachineControlPlane and MachineTypeInit here to maintain backwards compatibility with init-node style?

Reply:
:+1: